index.css

```diff
@@ -0,0 +1,154 @@
+@media (min-width: 400px) {
+  .header_container {
+    position: absolute;
+    width: 375px;
+    height: 92px;
```

Comment:
Please follow the guideline for all size of the component. The top bar is not the part of the project so no need to be 92px height.
<img width="412" alt="Screenshot 2022-12-02 at 2 09 58 AM" src="https://user-images.githubusercontent.com/25887253/205178473-d07c0311-34fe-4276-bb26-425ff1d30d61.png">